<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textedClass</class>
 <widget class="QMainWindow" name="textedClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Просмотр файлов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="txtFilter">
      <property name="toolTip">
       <string>Фильтры по имени файлов (разделены символом &quot;;&quot;)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeView" name="dirsView"/>
       <widget class="QListView" name="filesView"/>
      </widget>
      <widget class="QTextEdit" name="textView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actSave">
   <property name="icon">
    <iconset>
     <normaloff>../../img/save.png</normaloff>../../img/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить состояние текущей сессии</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset>
     <normaloff>../../img/close.png</normaloff>../../img/close.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Завершение работы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="icon">
    <iconset>
     <normaloff>../../img/info.png</normaloff>../../img/info.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="toolTip">
    <string>Информация о приложении</string>
   </property>
  </action>
  <action name="actShowAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../img/all.png</normaloff>../../img/all.png</iconset>
   </property>
   <property name="text">
    <string>Все файлы</string>
   </property>
   <property name="toolTip">
    <string>Отображать все отфильтрованные файлы затенёнными</string>
   </property>
  </action>
  <action name="actAutosave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../img/autosave.png</normaloff>../../img/autosave.png</iconset>
   </property>
   <property name="text">
    <string>Автосохранение при выходе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="texted.qrc"/>
 </resources>
 <connections/>
</ui>
